### PR TITLE
Update license, cleanup rubocop

### PR DESCRIPTION
**Pull request by fishnix**
All minor non-functional changes.  Update the copyright dates and cleanup some rubocop stuff.  This cookbook has been public for a while, I'd like to push it to our place on the chef supermarket.

### CODE REVIEW

File: providers/app.rb [hunk 1]
```diff
@@ -78,17 +78,17 @@ def create_app_dirs
...
-        action :create
-      end
-      new_resource.updated_by_last_action(a.updated_by_last_action?)
+    next if ::File.directory?(dir)
+
+    Chef::Log.info "Creating yajsw directory: #{dir}"
+    a = directory dir do
+      recursive true
+      owner appuser
+      action :create
```
Comment (cgatesman) [inline providers/app.rb]: Shouldn't this block of code be indented under the `next if`?

File: spec/unit/recipes/config_app_spec.rb [hunk 2]
```diff
@@ -35,14 +33,17 @@
...
-            expect(chef_run).to render_file('/etc/init.d/myapp').with_content(/^APP_HOME=\/usr\/local\/yajsw_apps\/myapp$/)
+            expect(chef_run).to render_file('/etc/init.d/myapp'
+                                ).with_content(%r{^APP_HOME=/usr/local/yajsw_apps/myapp$})
           end
         end
 
         it 'should render the yajsw wrapper config for the default app' do
-          expect(chef_run).to render_file('/usr/local/yajsw_apps/myapp/conf/wrapper.conf').with_content(/wrapper\.java\.app\.jar=lib\/com\.company\.myapp\.jar$/)
+          expect(chef_run).to render_file('/usr/local/yajsw_apps/myapp/conf/wrapper.conf'
+                              ).with_content(%r{wrapper.java.app.jar=lib/com.company.myapp.jar$})
```
Comment (cgatesman) [inline spec/unit/recipes/config_app_spec.rb]: In some places you line up `)` with the beginning of the method, and in other places you line it up with the matching `(`.  I'll never figure out you rubyists!

File: spec/unit/recipes/package_spec.rb [hunk 3]
```diff
@@ -21,8 +18,10 @@
         let(:install_script) { chef_run.remote_file('/usr/local/yajsw-stable-11.11.zip') }
 
         it 'should download the yajsw archive and notify the installation' do
-          expect(chef_run).to create_remote_file_if_missing('/usr/local/yajsw-stable-11.11.zip').with(owner: 'root')
-          expect(install_script).to notify('libarchive_file[extract_yajsw]').to(:extract).immediately
+          expect(chef_run).to create_remote_file_if_missing('/usr/local/yajsw-stable-11.11.zip'
+                                                           ).with(owner: 'root')
+          expect(install_script).to notify('libarchive_file[extract_yajsw]'
+                                          ).to(:extract).immediately
```
Comment (cgatesman) [inline spec/unit/recipes/package_spec.rb]: Case in point...

File: providers/app.rb (hunk 1 above)
Comment (fishnix) [inline providers/app.rb]: no,`next if ::File.directory?(dir)`  if is a complete statement

Comment (cgatesman) [inline providers/app.rb]: Then what does the following end match up with?

Comment (fishnix) [inline providers/app.rb]:
```
    a = directory dir do
      recursive true
      owner appuser
      action :create
    end
```

File: spec/unit/recipes/config_app_spec.rb (hunk 2 above)
Comment (fishnix) [inline spec/unit/recipes/config_app_spec.rb]: It's b/c of long lines.  I prefer to line them up with the opening `(` but then I end up over 100 characters in these lines, which is also a no-no.  The strings I'm matching on are too long but they don't really warrant a variable.

File: spec/unit/recipes/package_spec.rb (hunk 3 above)
Comment (fishnix) [inline spec/unit/recipes/package_spec.rb]: yeah see above ;)

File: providers/app.rb (hunk 1 above)
Comment (cgatesman) [inline providers/app.rb]: Oh.  Dah...  Thanks.

**Comment by cgatesman** [comment]
:+1:

**Comment by cjb06776** [comment]
:+1:

**Pull request merged**